packages/react-composites/src/composites/common/styles/ParticipantContainer.styles.ts

```diff
@@ -40,7 +40,7 @@ export const sidePaneHeaderStyles: IStackItemStyles = {
   root: {
     lineHeight: '1.5rem',
     fontSize: '1.125rem',
-    padding: '0.75rem 0.75rem 0.75rem 0.25rem',
+    padding: '0.75rem 0.75rem 0.75rem 0.50rem',
```

Comment:
Just to confirm this change is by design, the current figma's have it slightly out hence the current behavior:
![image](https://github.com/Azure/communication-ui-library/assets/2684369/af62c5e9-a1d7-4a80-aa8a-aca37d687b75)

Reply:
Seeing the same thing in Figma. Did Alex P file this bug?

Reply:
That is reported by AlexP in the bug bash, let me double check

Reply:
I talked to Alex, the Figma is updated with the alignment fix